.travis.yml

```diff
@@ -0,0 +1,32 @@
+---
+language: python
+python: "2.7"
+
+# Use the new container infrastructure
+sudo: false
+
+# Install ansible
+addons:
+  apt:
+    packages:
+    - python-pip
+    
+install:
+  # Install ansible
+  - pip install ansible
+
+  # Check ansible version
+  - ansible --version
+
+  # Create ansible.cfg with correct roles_path
+  - printf '[defaults]\nroles_path=../' >ansible.cfg
```

Comment:
printf does not supply a terminating newline.  Is that going to be an issue here?

Reply:
I borrowed this from another repo. The CI job seems to have completed successfully.